partials/base/base.html

```diff
@@ -3,6 +3,12 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <title>{{t "Firefox Accounts"}}</title>
+  <style>
+    .primary > span {
+      /* only affects the span's that are set to display:block; */
+      margin: 2px auto;
```

Comment:
We can't depend on styles working in a lot of mail clients, instead the styles need to be added to each span. Also a good reason to [surround the location with a span](https://github.com/mozilla/fxa-auth-mailer/pull/180#discussion_r71849114).